base/multi.jl

```diff
@@ -1479,6 +1481,7 @@ function addprocs_locked(manager::ClusterManager; kwargs...)
 
             if !isempty(launched)
                 wconfig = shift!(launched)
+                wconfig.enable_threaded_blas = Nullable{Bool}(params[:enable_threaded_blas])
```

Comment:
All the other fields of WorkerConfig are initialized in the managers only. This line should be moved here https://github.com/JuliaLang/julia/blob/6b6dead59db79f7940ad2d54d5269e5a319ed796/base/managers.jl#L212 and https://github.com/JuliaLang/julia/blob/6b6dead59db79f7940ad2d54d5269e5a319ed796/base/managers.jl#L323